evaluate.js

```diff
@@ -0,0 +1,98 @@
+const { flashHighlight, stringifyError } = require('./util');
+
+async function evalHighlighted(lang, selection, vscode, postWindow, editor) {
+  const selectionRange = new vscode.Range(
+    selection.start.line,
+    selection.start.character,
+    selection.end.line,
+    selection.end.character
+  );
+  const highlighted = editor.document.getText(selectionRange);
+
+  try {
+    const result = await lang.interpret(highlighted, null, true, false);
+    postWindow.appendLine(result.trim());
+    flashHighlight(vscode.window.activeTextEditor, selectionRange);
+  } catch (err) {
+    postWindow.appendLine(err);
+    console.error(err);
+  }
+}
+
+async function evalRegion(lang, vscode, postWindow, editor, hyperScopes) {
```

Comment:
instead of passin gin `hyperScopes` as a parameter:

`const hyperScopesExt = vscode.extensions.getExtension('draivin.hscopes');`

because you'll have your own `vscode` instance

Reply:
added here [7cf4cb3](https://github.com/alexander-daniel/scvsc/pull/12/commits/7cf4cb3f5935d564b471558808992ccff1ee67ae)